### DISCUSSION TITLE
Webhook to notify achievements unlocked

### DISCUSSION THREAD

**Discussion by krupupakku** [RetroAchievements/RAWeb#1199] [Ideas]
I'd like to add push notifications to my app and would be wonderful allow us to set a webhook and receive data every time a user unlocks an achievement (in my mind I imagine the webhook very similar as in slack, and the push notification like Microsoft does with xbox).

The idea is a user can receive push notification of their own progress only (maybe in the future we can think about adding friends) and it can be disabled if not desired, deleting the webhook. you can restore in setting in controlpanel in RAWeb.

**Comment by KP2048** [comment]
yes please. would like this for my steam deck plugin

**Reply by mirrorsedger** [reply]
Would be cool to have this implemented with the Emuchievements plugin for the Steam Deck.

**Reply by KP2048** [reply]
Literally what I was referring to (it's my plugin lol)

**Reply by mirrorsedger** [reply]
> Literally what I was referring to (it's my plugin lol)

Oh wow, I just recognised your name lmao

**Reply by KP2048** [reply]
Yea, otherwise it's kind of impossible to detect when an achievement has been unlocked and then trigger a reload of that games achievements data and display an achievement unlock toast

**Comment by manolakis** [comment]
Yes, this would be a great feature to integrate retroachevements into our apps!

**Reply by KP2048** [reply]
https://discord.com/channels/310192285306454017/1112459575069773964/1115622061356879903

**Reply by KP2048** [reply]
Apparently it's planned, but no eta
